mantle/cosa/build.go

```diff
@@ -70,3 +69,20 @@ func (build *Build) FindAMI(region string) (string, error) {
 	}
 	return "", fmt.Errorf("no AMI found for region %s", region)
 }
+
+func (build *Build) WriteMeta(path string, validate bool) error {
+	if validate {
+		valid, err := build.Validator()
+		if !valid {
+			return fmt.Errorf(
+				"refusing to render invalid metadata: %v",
+				err,
```

Comment:
Does this render the errors properly? I know in the past I've used the `multierror` package from `coreos/pkg` to have build a proper error string for a list of errors.